front50-core/src/main/groovy/com/netflix/spinnaker/front50/validator/PipelineValidator.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.front50.validator;
+
+import com.netflix.spinnaker.front50.model.pipeline.Pipeline;
+import org.springframework.validation.Errors;
+
+public interface PipelineValidator {
```

Comment:
I'd like to start seeing javadocs on all interfaces (class- & method-level) we introduce. This one is pretty explanatory, but it'll be a good habit to get into regardless.

Reply:
:thumbs_up: 

I like and can get behind that.